### DISCUSSION TITLE
Formatting ignoring `text-width`

### DISCUSSION THREAD

**Discussion by tim-ufer** [helix-editor/helix#8289] [General]
Hello, I'm using helix with Rust on an ultrawide screen so my tolerance for max line length is much higher than the default cut-off. I tried to extend the threshold by setting `text-width` to various high values in both my `config.toml` and `languages.toml` but when formatting files with either `:format` or auto-format on write, my longer lines still get hard-wrapped at the same width as before.
Other config changes take effect and I made sure to reload my config and restart the LSP but couldn't find out what I'm missing.

(Sorry if this was answered before; while searching I only found issues regarding soft-wrap)

**Comment by pascalkuthe** [comment]
Autoformatting hat nothing to do with text-width. Text-width is only used for softwrapping and `:reflow`.

Formatting is done by the languages servers or an external formatted binary. Both of which are external project with their own config. You should consult the documentation of whatever formatted you are using.
